services/getstream/deleteFeedById.js

```diff
@@ -2,9 +2,9 @@ const stream = require('getstream');
 const moment = require('moment');
 const GetstreamSingleton = require('../../vendor/getstream/singleton');
 
-module.exports = async (feedGroup, activityId, anonymousUserId) => {
+module.exports = async ({feedGroup, activityId, userId = '', token = ''}) => {
   const client = stream.connect(process.env.API_KEY, process.env.SECRET, process.env.APP_ID);
-  const feed = client.feed(feedGroup, anonymousUserId);
+  const feed = client.feed(feedGroup, userId, token);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The changes look good, but there's a potential issue with the default values for `userId` and `token`. If these are not provided, they will be set to empty strings. This could lead to unexpected behavior if the `client.feed()` function does not handle empty string inputs gracefully. It would be better to ensure that these parameters are always provided or handle the case where they are not.

```diff
- module.exports = async ({feedGroup, activityId, userId = '', token = ''}) => {
+ module.exports = async ({feedGroup, activityId, userId, token}) => {
```
If you want to keep the default values, consider adding error handling to check if `userId` and `token` are valid before passing them to `client.feed()`.


<!-- This is an auto-generated comment by OSS CodeRabbit -->